Portfolio desktop version

norbiboy1 opened the pull request:
- Align background professionally;
- Responsiveness and style for the two devices - mobile and desktop;
- Validation of contact form.

Commit:
Add:
- desktop version design with alignment all of the sections
- use grid and flexbox
- use media queries for the webpage
- validate contact form

Commit:
Changes for the desktop version:
- display flex and delete position: absolute on sections;
- add background-image and delete all the image sources;
- add active, hover, enable changing colors for all the buttons;
- fix the button disable icon(1 circle)

Commit:
I add a larger width for the desktop-list-languages.
I add width for the headline section.

Commit:
Add maximum width for html container to look responsive my webpage.

Commit:
Make my website looks professionaly by adding:
1. Media queries for each device - mobile and desktop.
2. Responsiveness.
3. Design and style.

norbiboy1 committed:
Merge branch 'main' into portfolio-desktop-version

Commit:
Resolve some errors with the displays on .css and .html file and linter configurations.

Qoosim reviewed (changes requested):
# *Status: Required Changes* :recycle: 

Hi @norbiboy1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Top Highlights

-  No linter errors. :heavy_check_mark: 
-  Good use of Github Flow. :heavy_check_mark: 
-  Good commit messages. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
![](https://camo.githubusercontent.com/38ce8c8b213b08e2d9bb85a0b79702ad512f3910f803ec3d7917b4e0110c9044/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966)


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Good job on implement some features to make your desktop version. You've really done great job. However, there are some issues you need to attend to before moving on. Your desktop implementation broke some of your mobile version. Kindly take a look at the below snapshots and fix all the areas where needed.

### Your Design hamburger for mobile

![your-hamburger-icon](https://user-images.githubusercontent.com/26597432/188266789-1c786cba-2fb6-477a-bde3-8b6f48dd8ce0.png)

#
### Figma Design hamburger

![figma-hamburger-icon](https://user-images.githubusercontent.com/26597432/188266805-3a284cde-828d-4ce4-b56c-23222afe0ed3.png)
 
#

### Your form design on mobile

- [ ] Kindly consider giving your form section some padding left and right so as to make it look great and inline with your chosen Figma template.

![form-design-yours](https://user-images.githubusercontent.com/26597432/188267042-6c14c06a-1617-49bb-b1a9-a8aa029fbb90.png)


#

### Figma form design on mobile

![form-design-figma](https://user-images.githubusercontent.com/26597432/188267003-86a764bb-d0a7-4608-8fdc-925d9fddf74c.png)

#

### Your footer Icons position on desktop

- [ ] Kindly consider styling your footer icons to be at the center of the page in order to be inline with the figma template you chose.

![your-footer-icons](https://user-images.githubusercontent.com/26597432/188267062-0ece9433-5b9f-432a-bf88-9ca5b2d5b5c6.png)

#

### Figma Design Icon position on desktop

![figma-footer-icons](https://user-images.githubusercontent.com/26597432/188267073-9bbabc9a-37a7-4924-8a1b-4a026968e7e5.png)

#

### Your design icons position on mobile

- [ ] Kindly consider reducing the space between your footer icons and your email address and leave some margin bottom to the section so as to be inline with the figma template you chose.

![your-footer-space](https://user-images.githubusercontent.com/26597432/188267214-32a7a47e-b7ea-43e8-b149-919e097993ed.png)

#

### Figma Design icons on mobile

![figma-footer-space](https://user-images.githubusercontent.com/26597432/188267226-7cea9a51-646f-4767-b101-24f901f86897.png)

Commit:
Add:
1. Resolving every issues found from the code reviewer.
2. Put div and substituite button by the animation event on it.

CollinsTatang commented on portfolio.css:
```diff
@@ -1,1025 +1,1825 @@
+  }
+}
+
+/* Media queries for the desktop version breakthrough */
+
+@media only screen and (min-width: 768px) and (max-height: 3846px) {
```
- [ ] Please kindly fix the mobile responsiveness as mentioned by the previous reviewer [Check Figmal Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
**Your Design**
![checkReview](https://user-images.githubusercontent.com/55596970/188272815-c498d677-75c2-46cf-a8ce-cf81b3a5a9bb.png)

**Figmal Design**
![image](https://user-images.githubusercontent.com/55596970/188272881-e1cb1b53-d254-4994-a800-e10f987bb329.png)

CollinsTatang reviewed (changes requested):
Hi @norbiboy1, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- [ ] Please kindly fix the mobile responsiveness as mentioned by the previous reviewer

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Add responsive design for the hamburger menu.

CollinsTatang reviewed (approved):
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
![image](https://user-images.githubusercontent.com/25227022/165483662-7b06c812-4cc0-40b9-a84c-4729d4cf10c3.gif)



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
